index.html

```diff
@@ -1396,6 +1396,40 @@ <h2>A project for first-time contributions</h2>
       </div>
       <!-- ________ Contributor's card END ________  -->
 
+      <!-- ________ *Monesh Goyal Card* ________  -->
+      <div class="card">
+        <p class="name">Monesh Goyal</p>
+        <p class="contact">
+          <i class="fab fa-twitter"></i>
+          <a href="www.twitter.com" target="_blank">@GoyalMonesh</a>
+        </p>
+        <p class="about">
+          Open Source enthusiast | Front-end Developement | Java
+        </p>
+        <div class="resources">
+          <p>3 Useful Dev Resources</p>
+          <ul>
+            <li>
+
```

Comment:
Please remove blank line.

Reply:
@lax0248917 Thanks for the suggestion working on the changes.